src/dstack/_internal/server/services/jobs/configurators/dev.py

```diff
@@ -32,13 +32,14 @@ def _shell_commands(self) -> List[str]:
         commands.append(INSTALL_IPYKERNEL)
         commands += self.run_spec.configuration.setup
         commands.append("echo ''")
+        commands += self.run_spec.configuration.init
+        commands.append("echo ''")
```

Comment:
`echo ''` now runs twice in a row if self.run_spec.configuration.init is empty, which breaks the tests. I guess the second `echo ''` is not needed if self.run_spec.configuration.init is empty.

Reply:
done!